WooCommerce/Classes/ViewRelated/JetpackSetup/JetpackSetupCoordinator.swift

```diff
@@ -323,16 +328,18 @@ private extension JetpackSetupCoordinator {
 
     func showMagicLinkUI(email: String) {
         analytics.track(event: .JetpackSetup.loginFlow(step: .magicLink))
-        let viewController = WPComMagicLinkHostingController(email: email, requiresConnectionOnly: requiresConnectionOnly)
+        let viewController = WPComMagicLinkHostingController(email: email,
+                                                             title: loginViewTitle,
+                                                             isJetpackSetup: true)
```

Comment:
Opinion: Similar to the previous comment, consider having a let constant to define this `isJetpackSetup` value.

Reply:
I'll update `JetpackSetupCoordinator` by injecting the login coordinator and the `isJetpackSetup` value there.